Two Pointers/three_sum.py

```
@@ -0,0 +1,74 @@
+            sum = a + nums[l] + nums[r]
+            if sum > 0:
+                r -= 1
+            
+            elif sum < 0:
+                l += 1
+            
+            else:
+                result.append([a, nums[l], nums[r]])
+                l += 1
+                while nums[l] == nums[l - 1] and l < r:     # To skip the duplicate values of 'l'
+                    l += 1
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)